setup.py

```diff
@@ -3,8 +3,7 @@
 
 ENTRY_POINTS = {
     'pipeline.handlers': [
-        'AatamsDmHandler = aodndata.aatams.aatams_dm:AatamsDmHandler',
-        'AatamsNrtHandler = aodndata.aatams.aatams_nrt:AatamsNrtHandler',
+        'AatamsSattagQcDmHandler = aodndata.aatams.aatams_sattag_qc_dm:AatamsSattagQcDm',
```

Comment:
the handler/folder should be renamed. There is no point keeping the DM part anymore, it's a 'generic' pipeline for both dm and nrt

Reply:
Yeap - completely forgot that I had to merge both logics into one pipeline - I think I should rebase then.